app/org/sagebionetworks/bridge/upload/UploadValidationTaskFactory.java

```diff
@@ -0,0 +1,39 @@
+package org.sagebionetworks.bridge.upload;
+
+import javax.annotation.Resource;
+import java.util.List;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import org.sagebionetworks.bridge.dao.UploadDao;
+import org.sagebionetworks.bridge.models.studies.Study;
+import org.sagebionetworks.bridge.models.upload.Upload;
+
+@Component
+public class UploadValidationTaskFactory {
```

Comment:
Discussed this offline with Alx. Having context.getBean() in my code means my unit tests have to be JUnit aware, which adds a bunch of complications to unit tests. The net complexity of using a factory object instead of calling context.getBean() is much lower, so I'm going to stick with the Factory class.